cycode/cyclient/scan_client.py

```diff
@@ -30,11 +30,13 @@ def __init__(
 
         self._hide_response_log = hide_response_log
 
-    def get_scan_controller_path(self, scan_type: str, should_use_scan_service: bool = False) -> str:
+    def get_scan_controller_path(
+        self, scan_type: str, should_use_scan_service: bool = False, should_use_cli_path: bool = False
+    ) -> str:
         if scan_type == consts.INFRA_CONFIGURATION_SCAN_TYPE:
             # we don't use async flow for IaC scan yet
             return self._SCAN_SERVICE_CONTROLLER_PATH
-        if not should_use_scan_service and scan_type == consts.SECRET_SCAN_TYPE:
+        if not should_use_scan_service and scan_type == consts.SECRET_SCAN_TYPE and not should_use_cli_path:
```

Comment:
can't we reuse the `should_use_scan_service` flag to achieve the same logic instead of adding a new one?

Reply:
@saramontif could we resolve this conversation?

Reply:
resolved